gooddata-sdk/gooddata_sdk/catalog/data_source/declarative_model/physical_model/pdm.py

```diff
@@ -0,0 +1,53 @@
+# (C) 2022 GoodData Corporation
+from __future__ import annotations
+
+from typing import Any
+
+from gooddata_metadata_client.model.declarative_model import DeclarativeModel
+from gooddata_metadata_client.model.declarative_tables import DeclarativeTables
+from gooddata_sdk.catalog.data_source.declarative_model.physical_model.table import CatalogDeclarativeTable
+
+
+class CatalogDeclarativePdm:
+    def __init__(self, pdm: CatalogDeclarativeTables = None):
+        self.pdm = pdm
+
+    @classmethod
+    def from_api(cls, entity: dict[str, Any]) -> CatalogDeclarativePdm:
+        pdm = CatalogDeclarativeTables.from_api(entity["pdm"])
+        return cls(pdm)
+
+    def to_api(self) -> DeclarativeModel:
+        kwargs: dict[str, Any] = dict()
+        if self.pdm:
+            kwargs["pdm"] = self.pdm.to_api()
+        return DeclarativeModel(**kwargs)
```

Comment:
typo DeclarativePdm and inDeclarativePdm you can pass pdm directly without using kwargs

Reply:
fixed